manifests/templates/release/cdi-operator-cr.yaml.in

```diff
@@ -2,3 +2,6 @@ apiVersion: cdi.kubevirt.io/v1alpha1
 kind: CDI
 metadata:
   name: cdi
+  namespace: {{.Namespace}}
```

Comment:
Are we expecting this name space to default to the 'cdi' namespace, since that is where we normally install CDI.

Reply:
We want to leave it configurable as other projects do